src/lib/scopes.ts

```diff
@@ -50,11 +50,13 @@ export function convertScopesToExternal (
     const accountId: string = scope.accountId
 
```

Comment:
what for `ExternalScope` is introduced ?

Reply:
there's a mismatch between the scope interface of the internal database and that received/sent out by handlers - as per our discussion with jj and lewis - we introduced to the `ExternalScope` interface for clarity